lib/ControlledVocab/tests/ControlledVocab-test.js

```diff
@@ -160,6 +164,22 @@ describe('ControlledVocab', () => {
       expect(cv.cancelButton).to.exist;
     });
 
+    describe('Cancel button is clicked', () => {
```

Comment:
Please rename it to look

```suggestion
    describe('Clicked Cancel button', () => {
```

Please check other places.